doc/env.md

```diff
@@ -1,8 +1,14 @@
 # Env
 
```

Comment:
```suggestion
Class `Napi::Env` inherits from class [`Napi::BasicEnv`][].

```

and then add

```
[`Napi::BasicEnv`]: ./basic_env.md
```

to the bottom of the file.

Reply:
Addressed in [195ec28](https://github.com/nodejs/node-addon-api/pull/1514/commits/195ec28a7b1dcfe571c6df5f8a2e1bb991126ff1).